sources/pmwebapi.go

```diff
@@ -110,7 +110,44 @@ func typeLabel(units string, pcpType string, name string) string {
 	} else {
 		name += unit
 	}
-	name = strings.Replace(name, "count / sec", "count_per_second", -1)
+	return name
+}
+
+func fixNaming(name string) string {
+	name = strings.ToLower(name)
+	switch name {
```

Comment:
This is forward progress, but I don't think this actually does what you expect it to do. Try testing with something like `disk_access_time_nanosec`. The result _should_ be `disk_access_time_nanoseconds`, but the way this is written, it will still be `disk_access_time_nanosec`. I will leave this as an exercise for you to figure out what's wrong, but let me know if you get absolutely stuck.